docs/components/buttons/android.md

```diff
@@ -2,25 +2,54 @@
 
 ## Syntax
 
+There are 2 options - button with pre-defined content and custom content. This showcases the pre-defined option with a text and an optional icon.
 ```kotlin example
 @Composable
 fun WarpButton(
-    text: String, 
-    onClick: () -> Unit, 
-    modifier: Modifier = Modifier, 
-    enabled: Boolean = true, 
-    buttonStyle: WarpButtonStyle, 
-    maxLines: Int = 1, 
-    loading: Boolean = false
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    buttonStyle: WarpButtonStyle = WarpButtonStyle.Primary,
+    maxLines: Int = 1,
+    loading: Boolean = false,
+    @DrawableRes icon: Int? = null,
+    iconContentDescr: String? = null
+)
+```
+To add custom content follow this syntax:
+```kotlin example
+@Composable
+fun WarpButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    buttonStyle: WarpButtonStyle = WarpButtonStyle.Primary,
+    loading: Boolean = false,
+    content: @Composable RowScope.() -> Unit
 )
 ```
 
 ## Visual options
 
+The button supports a variety of styles: 
+```kotlin example
+WarpButtonStyle {
+    Primary,
+    Secondary,
+    Quiet,
+    Negative,
+    NegativeQuiet,
+    Utility,
+    UtilityQuiet,
+    UtilityOverlay,
```

Comment:
@imprashast is this what the replacement of Button-pill will be called?

Reply:
Hmm, not sure? We never had a Button-Pill :D